### ISSUE TITLE
publish wheels on pypi

### ISSUE THREAD

**Issue by douardda** [jelmer/subvertpy#28]
it would be great to have python wheels published on pypi along the source package. Would help CI kind of tasks (especially tests in tox and the like).

**Comment by jelmer** [comment]
+1 If you're interested in working on this, help would be great.

**Comment by jelmer** [comment]
I think the easiest way to do this is to update the appveyor file to install the right dependencies; we already have it set up to build and publish msi and wheel files.

To a large extent, this means adding the steps documented in section E (Building the Latest Source under Windows) in appveyor.yml.

**Comment by k10blogger** [comment]
So i setup up the environment. After some struggle. I am getting the following syntax error for the `subvertpy-upstream-0.10.1` at util.c at [lines](https://github.com/jelmer/subvertpy/blob/6b99d9ba1393b52f107eb6b20b5d4a113e23c0d9/subvertpy/util.c#L1055-L1070) 
![image](https://user-images.githubusercontent.com/4973182/82708098-138b9480-9c9b-11ea-902e-70cfb5720c65.png)

Any idea how to resolve this? From my understanding it is assigment to existings variables.  So i guess a hopefull guess that `.<varible_name> = value,` should only be `value,`.

**Comment by jelmer** [comment]
Yeah, some fairly recent versions of MSVC don't support C99 style struct member assignments. Feel free to replace with ordered member values. Note that you can't just drop the ".foo = " bit, you'll need to make sure that they're in the right order as well and that any intermediate member variables are initialized.

**Comment by k10blogger** [comment]
I did that now i am stuck at the linking part seems some library mismatch can you help me with that.
From the picture below:
I could not find neon library in the 1.8.17 dev folder. or any 1.8.17 folder.
I removed the libintl completely
![image](https://user-images.githubusercontent.com/4973182/82710776-581a2e80-9ca1-11ea-9de4-1c5acc88bdcf.png)

From image further below
1 - I am able to find these libraries
2 - Removed the intl as not needed and optional
3 - found these in the VC for Python SDK folder
But i cannot find the zlibstat library. Do you know where to look for?
![image](https://user-images.githubusercontent.com/4973182/82710744-3de05080-9ca1-11ea-8fd1-857080e70a38.png)

**Comment by jelmer** [comment]
FWIW that's in the bit of code I have removed from subvertpy, because it's too crufty. I don't think it's useful to try and build subvertpy that way - I'd recommend just building subversion using their instructions and once you've done that, then try and build subvertpy.

**Comment by jelmer** [comment]
FWIW I've been moving my projects over to GitHub actions from appveyor/travis.

The .github/workflows/pythonpackage.yml code has the beginnings of this for subvertpy.

**Comment by k10blogger** [comment]
Thanks for the information. I am still stuck at building subversion on windows. Reading up on that.

**Comment by jelmer** [comment]
There is some progress towards this - now merged into master. 

The github action will actually build subvertpy on Windows. The modules load, but the testsuite currently crashes.

Once that's addressed, we could potentially use the github actions to publish a wheel on Windows.

**Comment by k10blogger** [comment]
Hi It seems from the build history 
https://github.com/jelmer/subvertpy/runs/1377423910?check_suite_focus=true

It fails at this action
https://github.com/jelmer/subvertpy/actions/runs/355021529/workflow#L110

msbuild needs a build package like the following on the system like this one
https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16

Need to read up on how to get it in the actions or the machine.

**Comment by jelmer** [comment]
Hmm, odd - that used to work - I've added the missing action to install msbuild.
